miden/Cargo.toml

```diff
@@ -54,10 +54,13 @@ structopt = { version = "0.3", default-features = false, optional = true }
 verifier = { package = "miden-verifier", path = "../verifier", version = "0.4", default-features = false }
 
 [dev-dependencies]
+assert_cmd = "2.0.7"
```

Comment:
small nit: we might wan't to follow the pattern of other crates and use only `major/minor` instead of `major/minor/patch`